### ISSUE TITLE
[Question] Support for Images?

### ISSUE THREAD

**Issue by tszumowski** [enjalot/latent-scope#24]
Hi! I have a question and it appears the Discord invite link is now invalid. 

Do you have plans to support showing images in the map? Either: 

(a) as part of the metadata, akin to what's shown for nomic.ai (in [this example](https://atlas.nomic.ai/map/stablediffusion))

<img width="861" alt="image" src="https://github.com/enjalot/latent-scope/assets/10282962/c29b10d6-7332-453b-bafd-37cba7e0e2f8">

or

(b) as part of the map icon's themselves, as shown in in TensorFlow Embedding Projector, the "MNIST with Images" example, here:
https://projector.tensorflow.org/.

<img width="940" alt="image" src="https://github.com/enjalot/latent-scope/assets/10282962/0defa321-df23-414d-bfa4-3940d9e41d06">

**Comment by enjalot** [comment]
Yes I'd love to support images! I think showing it in the metadata as in the first example you showed would be relatively straightforward, especially once #6 is done we could show thumbnails in the table if we detect an image url.

As for showing on the map, I do have some ideas for how that could be manageable, tho its often challenging depending on the size of the dataset. For larger sizes, doing a preprocessing step to generate tiles of thumbnails can be really helpful.

I actually want to support image embeddings in the future, using something like CLIP, and then we'd definitely want ways to show images on the map

Sorry about the discord link, i didn't realize they expire, here is a new one and i will update the README: https://discord.gg/x7NvpnM4pY

**Comment by enjalot** [comment]
I'm not showing images in the map yet, but they do show up in the table in 0.2.0
I will have to upload a tutorial. Showing images on the map may be a while down the road as it would mean another step to create some kind of sprite sheet for practical purposes

**Comment by enjalot** [comment]
Shipped. Image support landed across #124 (image embeddings) and #127/#130 (images on the map): for image datasets the map is a continuous level-of-detail — heatmap when zoomed out, a tiled representative-image sprite atlas as you zoom in, then individual points on top for hovering. Generated by the optional post-scope `ls-sprite-atlas` step.
